auth-web/src/components/auth/manage-business/AffiliationAction.vue

```diff
@@ -478,26 +521,22 @@ export default defineComponent({
       ![NrDisplayStates.HOLD, NrDisplayStates.EXPIRED, NrDisplayStates.PROCESSING].includes(status(item) as NrDisplayStates)
     }
 
-    // Refactor this is duplicated.
-    const isCurrentOrganization = (orgId: number) => {
-      return orgId === orgStore.currentOrganization.id
-    }
-
-    const isAffiliationInvitation = (item: Business): boolean => {
+    const showAffiliationInvitationCancelRequestButton = (item: Business): boolean => {
       return item.affiliationInvites?.length > 0 &&
-        item.affiliationInvites[0].status !== AffiliationInvitationStatus.Accepted
+        item.affiliationInvites[0].status !== AffiliationInvitationStatus.Accepted &&
```

Comment:
No cancel button for REQUEST? seems weird because in your code above it looks like it's possible

Reply:
Cancel is main action on REQUEST type